### ISSUE TITLE
InputStream Adaptive binary not loaded when play a video on Android

### ISSUE THREAD

**Issue by CastagnaIT** [xbmc/xbmc#23186]
### Describe the bug
Here is a clear and concise description of what the problem is:

On last nightlies of Kodi 21 from master branch
i noticed a super weird behaviour where Kodi is not able to load InputStream Adaptive binary when i try to play a video with it

this happens only when i try to play a video by browsing addon folders,
instead if i add the same video to favourites menu, and i play the video from favourites menu, then InputStream Adaptive binary is loaded with success and playback starts correctly

This is the log where i try to play the video by browsing the addon folders:
https://paste.kodi.tv/zatesepera.kodi

Just to comparing both behaviours, this is the log where i add the same video of log above to the favourites menu, then i played the video by using favourites menu:
https://paste.kodi.tv/epemedavuq.kodi

you can see that in the first log where the binary isnt loaded, so its missing:
```
info <general>: Creating InputStream
debug <general>: ADDON: Dll Initializing - InputStream Adaptive
```

## Expected Behavior
Here is a clear and concise description of what was expected to happen:

be able to play videos by using addons folders

## Actual Behavior

kodi dont load InputStream binary

## Possible Fix

no idea

### To Reproduce

1. Install an addon that use InputStream adaptive to play videos (i used N€tflix but same behaviour with "Rai Play" addon)
2. Browse the addon and play the video
3. Playback never start

### Debuglog

The debuglog can be found here:
linked above

### Screenshots 
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [x] Android
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [ ] macOS
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Nvidia shield 9.1.1
 - Kodi version: master branch

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by joseluismarti** [comment]
I think something has been accidentally introduced in #23114, in particular in 728b396dad

Test a build with:
```
git revert 6df4005f83 2b4559e2ee f22fa41ec5 728b396dad
```

**Comment by neo1973** [comment]
@CastagnaIT @joseluismarti Did your tests include this fix from yesterday? #23181

**Comment by neo1973** [comment]
Found the issue. PR is on its way.

**Comment by CastagnaIT** [comment]
fixed on #23190
